[2023-11-23T22:12:26.225+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-23T22:12:26.242+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-23T22:12:26.243+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-23T22:12:26.258+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-20 00:00:00+00:00
[2023-11-23T22:12:26.263+0000] {standard_task_runner.py:57} INFO - Started process 139 to run task
[2023-11-23T22:12:26.267+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpwkyoc_rc']
[2023-11-23T22:12:26.271+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask extract
[2023-11-23T22:12:26.326+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [running]> on host 3c6bdd9606b4
[2023-11-23T22:12:26.692+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-23T22:12:27.287+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-23T22:12:27.299+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=extract, execution_date=20231120T000000, start_date=20231123T221226, end_date=20231123T221227
[2023-11-23T22:12:27.332+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T22:12:27.352+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-24T21:55:09.600+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-24T21:55:09.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-24T21:55:09.613+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-24T21:55:09.798+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-20 00:00:00+00:00
[2023-11-24T21:55:09.805+0000] {standard_task_runner.py:57} INFO - Started process 199 to run task
[2023-11-24T21:55:09.810+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpnywkbhe8']
[2023-11-24T21:55:09.815+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask extract
[2023-11-24T21:55:09.871+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [running]> on host 30eca782d34c
[2023-11-24T21:55:09.955+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-24T21:55:10.843+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_scripts/extract.py", line 35, in extract_data
    storage_client = storage.Client('oceanic-hangout-406022')
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/storage/client.py", line 172, in __init__
    super(Client, self).__init__(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/client/__init__.py", line 321, in __init__
    Client.__init__(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/client/__init__.py", line 178, in __init__
    credentials, _ = google.auth.default(scopes=scopes)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 657, in default
    credentials, project_id = checker()
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 650, in <lambda>
    lambda: _get_explicit_environ_credentials(quota_project_id=quota_project_id),
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 270, in _get_explicit_environ_credentials
    credentials, project_id = load_credentials_from_file(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 114, in load_credentials_from_file
    raise exceptions.DefaultCredentialsError(
google.auth.exceptions.DefaultCredentialsError: File opt/***/dags/etl_scripts/oceanic-hangout-406022-fd958fa4d302.json was not found.
[2023-11-24T21:55:10.861+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=extract, execution_date=20231120T000000, start_date=20231124T215509, end_date=20231124T215510
[2023-11-24T21:55:10.876+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 12 for task extract (File opt/***/dags/etl_scripts/oceanic-hangout-406022-fd958fa4d302.json was not found.; 199)
[2023-11-24T21:55:10.912+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-24T21:55:10.933+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-24T23:01:47.767+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-24T23:01:47.815+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-24T23:01:47.820+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-24T23:01:47.892+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-20 00:00:00+00:00
[2023-11-24T23:01:47.993+0000] {standard_task_runner.py:57} INFO - Started process 105 to run task
[2023-11-24T23:01:47.997+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpnv8jlobx']
[2023-11-24T23:01:48.007+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask extract
[2023-11-24T23:01:48.148+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-20T00:00:00+00:00 [running]> on host 69a30b58663a
[2023-11-24T23:01:48.498+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-24T23:01:49.586+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-24T23:01:49.607+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=extract, execution_date=20231120T000000, start_date=20231124T230147, end_date=20231124T230149
[2023-11-24T23:01:49.677+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-24T23:01:49.724+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
